### ISSUE TITLE
External link icon description is not correct 

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/Icons#34]
In this following code, we are making a statement that is incorrect. 

https://github.com/AlaskaAirlines/Icons/blob/2d46a1e7970f707224978aa6686c38ecac6e3ddb/src/data/icons.json#L237

> This indicates a link to an external site that may not follow the same accessibility or privacy policies as Alaska Airlines. By selecting a partner link you agree to share your data with these sites.

According to the [specification](https://www.w3.org/TR/WCAG20-TECHS/G201.html), the use of this icon is not only for external links. 

> The objective of this technique is to provide a warning before automatically opening a new window or tab. Opening new windows automatically when a link is activated can be disorienting for people who have difficulty perceiving visual content, and for some people with cognitive disabilities, if they are not warned in advance. Providing a warning allows the user to decide it they want to leave the current window, and the warning will help them find their way back, if they do decide they would like to go to the new window. It will help them understand that the "back" button will not work and that they have to return to the last window they had open, in order to find their previous location.

The content for this icon should be revised to accurately speak to its intent. 

It is my opinion that this UI icon was co-opted for the use of the CCPA and other privacy concerns. This is not the intent of the icon, but there is much debate as to what is. In this issue, I will try to provide as many resources and references as possible to allow for a reasonable conclusion.

**Comment by blackfalcon** [comment]
## Links to New Windows, Pop-ups, Other Frames, or External Web Sites

There is much debate about the merits of links that open in new windows, pop-up windows, or other frames. Some people would like to ban them entirely. Others concede that they can be appropriate at times, but nearly everyone agrees that users ought to be alerted when the link does not open in the current window or frame. The accessibility issue is that some users can get confused with the new windows or tabs. Newer screen readers alert the user when a link opens a new window, though only after the user clicks on the link. Older screen readers do not alert the user at all. Sighted users can see the new window open, but users with cognitive disabilities may have difficulty interpreting what just happened. Then when the try to click on the Back button in the browser, nothing happens, because there is no previous link to go back to in a new window or tab.

As with links to non-HTML files, browsers could potentially perform the function of alerting users, but current browsers do not perform this functionality. It is up to the authors to alert the users. Authors can add this information to links by placing it in parenthesis at the end of the link, for example by saying "Biographical sketch (opens new window)." Some sites use icons to convey this same message. Some of the icons used by various web sites appear below.

![Example icons](https://user-images.githubusercontent.com/181089/88207856-3d165d00-cc05-11ea-9128-bdf4f1928dbb.png)

**Sometimes these icons are used to indicate that a link is to an external site, rather than that it opens a new window, so there's potential for some confusion. As with file types above, these icons (with appropriate alternative text) must appear within the link rather than just after it.**

[Links and Hypertext | WebAIM](https://webaim.org/techniques/hypertext/hypertext_links)

**Comment by blackfalcon** [comment]
Another example found is from 
https://www.ahrq.gov/externaldisclaimer.html

This disclaimer only addresses the issue of leaving this Federal Government Web site, but uses a similar icon that is used with opening a new window.

**Comment by blackfalcon** [comment]
It might be interesting to note that https://www.deque.com/axe/ a leader in a11y has links on this page that take the user to a new domain with no disclaimer at all that this will happen. 

Example: `Web Testing API` goes to https://github.com/dequelabs/axe-core

**Comment by blackfalcon** [comment]
https://www.cdc.gov/other/disclaimer.html

Here is a well-documented position from the CDC. There use is very interesting and something that I would consider best practice. 

The same icon is used for any case where the user will be redirected to a URL other than the domain they are currently on. But, in a case where the action will open a new tab there is an Exit Disclaimer dialog that the user will interact with. 

Clicking CANCEL keeps the user from opening a new tab. Clicking OK allows the tab action to happen and this also sets a cookie so that any link with that same URL and action will not produce the dialog.

**Comment by blackfalcon** [comment]
This issue is tied directly to https://github.com/AlaskaAirlines/auro-hyperlink/issues/44

**Comment by blackfalcon** [comment]
https://github.com/AlaskaAirlines/auro-hyperlink/pull/64

**Comment by blackfalcon** [comment]
This issue was addressed with this release https://github.com/AlaskaAirlines/auro-hyperlink/pull/64
